### PR TITLE
Support parsing live video URLs

### LINKED ISSUE #740: Using a live video URL throws an exception
### Version

v6.3.4

### Platform

.NET 7 / Windows 11 / Ubuntu

### Steps to reproduce

- Search YouTube for a live stream
- Go live with the results
- Click on Share
- See that the URL shown to copy is similar to https://www.youtube.com/live....
- Instantiate the YoutubeClient class
- Use the following standard code to get the video
```cs
var youtube = new YoutubeClient();
var video = await youtube.Videos.GetAsync("https://www.youtube.com/live/jfKfPfyJRdk?si=8-bqIN016D3rQT2x");
```

- The Exception below is generated
```
System.ArgumentException: 'Invalid YouTube video ID or URL 'https://www.youtube.com/live/jfKfPfyJRdk?si=8-bqIN016D3rQT2x'.'
```

### Checklist

- [X] I have looked through existing issues to make sure that this bug has not been reported before
- [X] I have provided a descriptive title for this issue
- [X] I have made sure that that this bug is reproducible on the latest version of the package
- [X] I have provided all the information needed to reproduce this bug as efficiently as possible
- [ ] I have sponsored this project

### PR DESCRIPTION
**Pull request by eimigueloliveir**
Close #740

### CODE REVIEW

File: YoutubeExplode/Videos/VideoId.cs [hunk 1]
```diff
@@ -73,6 +73,15 @@ public partial struct VideoId
...
 
+        // Live Share URL
+        // https://www.youtube.com/live/jfKfPfyJRdk
+        var liveShareMatch = Regex
+            .Match(videoIdOrUrl, @"youtube\..+?/live/(.*?)(?:\?|&|/|$)")
+            .Groups[1].Value.Pipe(WebUtility.UrlDecode);
+
+        if (!string.IsNullOrWhiteSpace(liveShareMatch) && IsValid(liveShareMatch))
+            return liveShareMatch;
+
```
Comment (Tyrrrz) [inline YoutubeExplode/Videos/VideoId.cs:84]:
```suggestion
        // Live URL
        // https://www.youtube.com/live/jfKfPfyJRdk
        var liveMatch = Regex
            .Match(videoIdOrUrl, @"youtube\..+?/live/(.*?)(?:\?|&|/|$)")
            .Groups[1].Value.Pipe(WebUtility.UrlDecode);

        if (!string.IsNullOrWhiteSpace(liveMatch) && IsValid(liveMatch))
            return liveMatch;

```

**Comment by Tyrrrz** [comment]
Thanks! Can you also please add tests here: https://github.com/Tyrrrz/YoutubeExplode/blob/master/YoutubeExplode.Tests/VideoIdSpecs.cs

**Comment by Tyrrrz** [comment]
Thanks!

**Comment by eimigueloliveir** [comment]
😀

**Pull request merged**